src/docs/user/references/DataConnectors.adoc

```diff
@@ -234,64 +234,85 @@ In the import mode, the output containing the imported data takes one or many of
+This feature is showcased through two workflow templates: SQL_Pooled_Connection_Query and SQL_Pooled_Connection_Update. Both are a single task workflow.
 
 *Variables:*
 
-Both workflows share the same list of variables. Consequently, we describe them in the same following table.
+The variables listed in the following table exist for both the SQL_Pooled_Connection_Update and the SQL_Pooled_Connection_Query workflows.
 
 .Workflows' variables
 [cols="2,5,2,2,2,3"]
 |===
 | *Variable name* | *Description* | *Scope* | *Required?*  | *Type*  | *Default/Examples*
 | `RDBMS_NAME`
 | Name of the relational database management system.
-| Workflow, Open_Pooled_Connection Task
+| Workflow, Task
 | Yes
 | String
-| PostgreSQL, MySQL, Greenplum, HSQLDB, Oracle
+| Values: `{PostgreSQL, MySQL, Greenplum, HSQLDB, Oracle}`
 | `HOST`
 | Server host.
-| Open_Pooled_Connection Task
+| Workflow, Task
 | Yes
 | String
 | `localhost`
 | `PORT`
 | Listening port.
-| Open_Pooled_Connection Task
+| Workflow, Task
 | No
 | Integer
 | e.g. 5432, 1521
 | `USERNAME`
 | Username to use for connecting to the database.
-| Open_Pooled_Connection Task
+| Workflow, Task
 | Yes
 | String
 | e.g MY_USER
 | `DATABASE`
 | Database name.
-| Open_Pooled_Connection Task
+| Workflow, Task
 | Yes
 | String
 | e.g. MY_DATABASE
 | `POOL_PROPERTY`
 | HikariCP framework provides many data source properties that can be used to configure the DB pooled connection such as autoCommit, maximumPoolSize, maxLifetime, idleTimeout .... You can add as many properties as you want. For each one, please add a new task variable where the variable name is the property name having "POOL_"  as a prefix and the variable value is the property value. For more info, please refer to https://github.com/brettwooldridge/HikariCP.
-| Open_Pooled_Connection Task
+| Workflow, Task
 | No
 | String
 | e.g. POOL_autoCommit, POOL_maximumPoolSize
 | `SQL_STATEMENTS`
 | To change or update data, you have to use CREATE, INSERT , UPDATE , or DELETE statements. To query data, you have to perform SELECT statements.
-| Execute_Query_Statements Task, Execute_Update_Statements Task
+| Workflow, Task
 | Yes
 | String
 | e.g. ```SELECT * FROM ...```, ```DROP TABLE ...```
 |===
 
+In addition to the aforementioned variables, SQL_Pooled_Connection_Query workflow has two more variables.
+
+.SQL_Pooled_Connection_Query additional variables
+[cols="2,5,2,2,2,3"]
+|===
+| *Variable name* | *Description* | *Scope* | *Required?*  | *Type*  | *Default/Examples*
+| `OUTPUT_TYPE`
+| If set to `HTML`, it allows to preview the results in the Scheduler Portal in a HTML format. If set to `CSV`, it allows to download the results from the Scheduler Portal in a CSV format.
+| Workflow, Task
+| Yes
+| String
+| Values: `{CSV, HTML}`.
+
+Default: `HTML`
+| `STORE_RESULT_VARIABLE`
+| if not empty, the value will be the name of a variable that contains the resultSet (converted into a List of Maps) of the query. This variable can be used in other tasks.
```

Comment:
the resultSet of the query (converted into a List of Maps)? maybe better for readability